### PR TITLE
spring festival updates (fixed climber code so it doesn't break the r…

### PR DESCRIPTION
**Pull request by ambercmerrill**
…obot)

### CODE REVIEW

**Comment by edelmanjm** [comment]
You should probably be made a team admin now that you're the manager. Someone should still review it anyways tho.

File: AlternativeRobot/src/org/team1540/robot2017/commands/betterauto/MovementCommand.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package org.team1540.robot2017.commands.betterauto;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class MovementCommand extends Command {
+
+    @Override
+    protected boolean isFinished() {
+        // TODO Auto-generated method stub
```
Comment (edelmanjm) [inline AlternativeRobot/src/org/team1540/robot2017/commands/betterauto/MovementCommand.java:9]: boi

File: AlternativeRobot/src/org/team1540/robot2017/commands/JoystickClimb.java [hunk 2]
```diff
@@ -15,7 +15,7 @@ public JoystickClimb() {
     // Called repeatedly when this Command is scheduled to run
     @Override
     protected void execute() {
-        Robot.climber.set(RobotUtil.deadzone(OI.getClimberJoystick(), 0.2));
+        Robot.climber.set(RobotUtil.deadzone(Math.abs(OI.getClimberJoystick()), 0.2));
```
Comment (edelmanjm) [inline AlternativeRobot/src/org/team1540/robot2017/commands/JoystickClimb.java:18]: Shouldn't the climber be able to run backwards?

Comment (ambercmerrill) [reply AlternativeRobot/src/org/team1540/robot2017/commands/JoystickClimb.java:18]: no

Comment (edelmanjm) [reply AlternativeRobot/src/org/team1540/robot2017/commands/JoystickClimb.java:18]: but y

Comment (ambercmerrill) [reply AlternativeRobot/src/org/team1540/robot2017/commands/JoystickClimb.java:18]: it's a ratchet. if it runs backwards it'll break the robot

**Comment by JMY1000** [comment]
bruh u just review your own pr?

**Comment by edelmanjm** [comment]
Did you just review your own PR...

**Pull request merged**